java-client/build.gradle.kts

```diff
@@ -133,7 +133,7 @@ dependencies {
 
     val opensearchVersion = "2.1.0"
     val jacksonVersion = "2.13.3"
```

Comment:
Please also update Jackson version

Reply:
@reta this is bot PR :)

Reply:
Cr@p :) Thanks @saratvemulapalli , I expected `bot` somewhere in the name ... :-)